artemis-server/src/test/resources/ConfigurationTest-xinclude-config-acceptors.xml

```diff
@@ -14,7 +14,10 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<acceptors xmlns="urn:activemq:core">
+<acceptors xmlns="urn:activemq:core"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="urn:activemq:core ../../../../artemis-server/src/main/resources/schema/artemis-configuration.xsd"
```

Comment:
-1 on this should not be needed to be so explicit. this is very fragile

Reply:
Correct, none of these test files need to be so explicit. The {{broker-connections}} file element covers the original include namespace attribute test case matching the documentation.

I added the common namespace attribute combinations to the existing xinclude element test files to ensure adequate testing of includes with the updated XSD. This file and the {{address-settings}} include explicitly define the schema definition location, just as {{ConfigurationTest-xinclude-config.xml}} does, as well as an explicit type reference attribute. The {{addresses}} file skips the explicit type reference. The {{broker-connections}} file include sticks with the original, bare {{xmlns}} URN.